FHEM/00_SIGNALduino.pm

```diff
@@ -4936,11 +4934,12 @@ sub SIGNALduino_githubParseHttpResponse($$$)
 		<li>hardware<br>
     		When using the flash command, you should specify what hardware you have connected to the usbport. Doing not, can cause failures of the device.
 		<ul>
-			<li>ESP_1M: ESP8266 with 1 MB flash and CC1101 receiver</li>
 			<li>ESP32: ESP32</li>
-			<li>nano: Arduino Nano 328 with cheap receiver</li>
-			<li>nanoCC1101: Arduino Nano 328 wirh CC110x receiver</li>
+			<li>ESP8266: ESP8266 cheap receiver</li>
```

Comment:
Dass 1 MB Flash benötigt wird, sollten wir hinerlegen denke ich.

Reply:
Du meinst, du möchtest es ggf so

ESP8266: ESP8266 cheap receiver (with 1MB Flash)

Reply:
Zuerst würde ich mal vorschlagen, von dem Wort "billig" abzusehen. Stattdessen könnte man vieleicht "analog" verwenden.
Den Zusatz beim ESP8266 "with 1MB Flash" sollten wir ändern zu "mindestens 1 Mb Flash".

Reply:
@elektron-bbs 
Ich hab das noch ein wenig umformuliert und präzisiert. Schaust Du es dir bitte einmal an?

Reply:
Wenn ich ganz ehrlich sein soll, "einfacher eindraht Empfänger" trifft es auch nicht so richtig. Das kann ja auch ein aufwändiger Superhet statt eines einfachen Audion sein.
In einem Katalog fand ich jetzt auf die Schnelle: "On Data Output" - das trifft es sicher besser.

Reply:
Okay. One Data Output trifft es besser. Das Einfach kam mir im Vergleich zum cc1101 halt in den Sinn :)